<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>townsetClass</class>
 <widget class="QDialog" name="townsetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Циферблаты</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../timezones.qrc">
    <normaloff>:/res/k.ico</normaloff>:/res/k.ico</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTableWidget" name="twTowns">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column/>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pbAdd">
       <property name="toolTip">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="../timezones.qrc">
         <normaloff>:/res/plus.png</normaloff>:/res/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDel">
       <property name="toolTip">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="../timezones.qrc">
         <normaloff>:/res/editdelete.png</normaloff>:/res/editdelete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbUp">
       <property name="toolTip">
        <string>Up</string>
       </property>
       <property name="icon">
        <iconset resource="../timezones.qrc">
         <normaloff>:/res/uparrow.png</normaloff>:/res/uparrow.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbDown">
       <property name="toolTip">
        <string>Down</string>
       </property>
       <property name="icon">
        <iconset resource="../timezones.qrc">
         <normaloff>:/res/downarrow.png</normaloff>:/res/downarrow.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="toolTip">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../timezones.qrc">
         <normaloff>:/res/save.png</normaloff>:/res/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit">
       <property name="toolTip">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../timezones.qrc">
         <normaloff>:/res/exit.png</normaloff>:/res/exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../timezones.qrc"/>
 </resources>
 <connections/>
</ui>
